packages/libs/eda/src/lib/core/utils/default-axis-range.ts

```diff
@@ -29,50 +30,33 @@ export function numberDateDefaultAxisRange(
                   defaults.displayRangeMin <= 0 ||
                   defaults.rangeMin <= 0)
                   ? (observedMinPos as number)
-                  : observedMin != null
-                  ? Math.min(
-                      // add zero as an origin
-                      0,
+                  : (min([
+                      isNonZeroBaseline ? undefined : 0,
```

Comment:
nice

Reply:
maybe also leave a comment saying that lodash.min is being used because it is tolerant of undefined values